src/server/api/controllers/users.controller.js

```diff
@@ -1,6 +1,27 @@
 const knex = require('../../config/db');
 
-async function getUserBySlackId(slackId) {
+const getUsers = async () => {
+  try {
+    const users = await knex('users')
+      .select(
+        'users.id',
+        'users.name',
+        'slack_id',
```

Comment:
Always rename `_` to the next uppercase letter.
This becomes `slackId`. Same for the next two properties.